hello-app-tls/main.go

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Why the Javadoc style block comment?

More common is just prefixing each line with `// ` but this is getting really nitpicky.